slider/beatmap.py

```diff
@@ -1871,6 +1992,24 @@ def ar(self,
 
         return ar
 
+    @lazyval
+    def breaks(self):
+        """The breaks of this beatmap.
+        """
+        return tuple(e for e in self.events if isinstance(e, Break))
+
+    @lazyval
+    def backgrounds(self):
+        """The backgrounds of this beatmap.
+        """
+        return tuple(e for e in self.events if isinstance(e, Background))
```

Comment:
is it possible for there to be more than one background event? the previous code here assumed there would only be at most one background event, but reading the osu file format docs, nothing implies that only one such event can exist.